docs/accessibility_principles.md

```diff
@@ -0,0 +1,20 @@
+# Accessibility principles
+
+The latest version of the WCAG is WCAG 2.1, which defines some guidelines for covering accessibility on mobile devices. Still, it does not cover how to natively satisfy those guidelines on modern platforms like Android and iOS.
+
+Because of that, we defined our guidelines to help mobile developers to satisfy most of the guidelines defined by WCAG.
+
+In the following chapters, information about satisfying accessibility guidelines will be provided with descriptions and techniques:
+
+* [Perceivable principle](principles/perceivable_principle.md)
+* [Operable principle](principles/operable_principle.md)
+* [Understandable principle](principles/understandable_principle.md)
+* [Robust principle](principles/robust_principle.md)
```

Comment:
I would recommend to add `and guidelines`. Because readers would not think that with "principle" only you'll get to guidelines for every principle.

Reply:
How about if I remove the word `principle`, so it will be:
* Perceivable
* Operable
* Understandable
* Robust

I think having `XYZ principle and guidelines` is too much text.

Reply:
In that case, I would go with e.g. Perceivable guidelines, so just `<category> guidelines`.